nextflow.config

```diff
@@ -57,41 +42,36 @@ params {
     itsonedb_download_taxdump = "https://ftp.ebi.ac.uk/pub/databases/ena/taxonomy/sdwca.zip"
     itsonedb_tax_header = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/itsonedb_tax_header.txt"
 
-    max_memory                 = '128.GB'
-    max_cpus                   = 16
-    max_time                   = '240.h'
+    uniref90_version = "2024_05"
+    rheadb_version = "135"
+    uniprotKB_access_date = "2024-07-31"
+    uniref90_download_fasta = "ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref90/uniref90.fasta.gz"
+    rhea_chebi_download_mapping = "https://ftp.expasy.org/databases/rhea/txt/rhea-reactions.txt.gz"
```

Comment:
Have you tried: "ftp://ftp.expasy.org/databases/rhea/txt/rhea-reactions.txt.gz" instead?

Reply:
Yep, it solved the problem!
Thank you

Reply:
excellent :)

Reply:
Fixed in [download with file() instead of wget](https://github.com/EBI-Metagenomics/taxdb_generation_nf/pull/1/commits/ba9bdb311d0c9e080bad30f55eb4a0fa333f1372)